Tutorials/ASP.NET/Blazor.ServerSide/CS/BlazorServerSideApplication.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="DevExpress.Blazor" Version="20.1.6" />
-    <PackageReference Include="DevExpress.Xpo" Version="20.1.6" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="DevExpress.Blazor" Version="25.1.4" />
+    <PackageReference Include="DevExpress.Xpo" Version="25.1.4" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
I recommend you get rid of Newtonsoft.Json at since we don't use it in DX products anymore. Feel free to ask me or Dmitry Minakov if you need an assistance with it.